torch/csrc/autograd/functions/basic_ops.h

```diff
@@ -30,6 +34,10 @@ struct DelayedError : public Function {
 
   virtual variable_list apply(const variable_list& inputs) override;
 
+  virtual variable_list operator()(const variable_list& inputs) {
```

Comment:
I think this was causing the bug. The base method was doing the regular check, which doesn't hold here presumably.

Reply:
But this doesn't seem to work, and I am confused.